### ISSUE TITLE
Cards: Z-Index positioning for Social Media

### ISSUE THREAD

**Issue by eng-web** [eng-web/princeton-eng#25]
Created by Josh Cartagena via monday.com integration. 🎉

**Comment by eng-web** [comment]
- Entire card should be clickable
- Social media icons should be clickable

**Comment by anhek** [comment]
Hello, can I give a try on this one ? :)

**Comment by eng-web** [comment]
Thanks for reaching out @anhek, though we don't accept contributions at this time. Apologies for the confusion.
